repose-aggregator/components/filters/saml-policy-translation-filter/src/main/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilter.scala

```diff
@@ -20,22 +20,201 @@
 
 package org.openrepose.filters.samlpolicy
 
+import java.io.InputStream
 import javax.inject.{Inject, Named}
-import javax.servlet.{FilterChain, ServletRequest, ServletResponse}
+import javax.servlet.http.HttpServletResponse.SC_INTERNAL_SERVER_ERROR
+import javax.servlet.http.{HttpServletRequest, HttpServletResponse}
+import javax.servlet.{FilterChain, ServletInputStream, ServletRequest, ServletResponse}
+import javax.ws.rs.core.MediaType
 
+import com.google.common.cache.{Cache, LoadingCache}
```

Comment:
The `Cache` class does not seem strictly necessary at this point. The import seems to be unused.